plugins/wmap.rb

```diff
@@ -127,11 +127,14 @@ def cmd_wmap_sites(*args)
       while (arg = args.shift)
         case arg
         when '-a'
-          s = add_web_site(args.shift)
-          if s
-            print_status("Site created.")
-          else
-            print_error("Unable to create site")
+          site = args.shift
+          if site
+            s = add_web_site(site)
+            if s
+              print_status("Site created.")
+            else
+              print_error("Unable to create site")
+            end
```

Comment:
... else print_error...

Otherwise you're silently failing without telling the user. Thanks for catching this though, not sure I've ever tried that particular invocation :)

Reply:
Error message added. When I did so, I noticed some formatting inconsistencies throughout the plugin. I'll make a separate PR for this later (to keep the git history tidy).